docs/hooks/collections.mdx

```diff
@@ -75,6 +75,7 @@ import { CollectionBeforeOperationHook } from 'payload/types'
 const beforeOperationHook: CollectionBeforeOperationHook = async ({
   args, // original arguments passed into the operation
   operation, // name of the operation
+  req, // full express request
```

Comment:
This should specify that it is a `PayloadRequest`, not expess. Also, can we make these proper JSDoc comments that will show in the IDE?

Reply:
Nope, this is just documentation. We're in an `.mdx` here.